assets/src/setup/index.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * WordPress dependencies
+ */
+import { render } from '@wordpress/element';
+import domReady from '@wordpress/dom-ready';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * External dependencies
```

Comment:
This is an internal dependency, not a third-party package.
```suggestion
 * Internal dependencies
```

Reply:
Unfortunately ESLint won't allow this change because the import is interpreted as external, but I added a comment to the import to make sure it's clear where it's coming from.